app/src/main/res/authors.txt

```diff
@@ -44,6 +44,9 @@ panorama_path_surface_unpav... CC-BY 4.0         https://github.com/westnordost/
 panorama_surface_paved.jpg     CC0               https://www.goodfreephotos.com/united-states/indiana/hoosier-hill/hooiser-hill-road-near-the-top-indiana.jpg.php
 panorama_surface_unpaved.jpg   CC-BY 2.0         https://www.flickr.com/photos/ian_munroe/3633361321
 panorama_surface_ground.jpg    CC-BY 3.0         https://commons.wikimedia.org/wiki/File:Dirt_Road,_Bulacan,_Philippines_-_panoramio.jpg
```

Comment:
Any reason for keeping the panorama images?

Reply:
No, this images will be removed once I extend this to all other surfaces quests.